src/crimson/osd/osd.cc

```diff
@@ -99,7 +99,9 @@ OSD::OSD(int id, uint32_t nonce,
       update_stats();
     }},
     asok{seastar::make_lw_shared<crimson::admin::AdminSocket>()},
-    osdmap_gate("OSD::osdmap_gate", std::make_optional(std::ref(shard_services)))
+    osdmap_gate("OSD::osdmap_gate", std::make_optional(std::ref(shard_services))),
+    log_client(shard_services.get_cct(), cluster_msgr.get(), &monc->monmap, LogClient::NO_FLAGS),
```

Comment:
i'd avoid using `cct` in crimson unless we have to.

Reply:
> i'd avoid using `cct` in crimson unless we have to.

How would you suggest to replace it? it's mainly used for `lderr`, `ldout`.
Is the `logger()` an appropriate substitute?
`cct->_conf` can be replaced with crimson's `local_conf()`.